### PR TITLE
Standalone

### PR DESCRIPTION
**Pull request by jbygdell**
* Enable standalone deployment
* Test deployment of standalone

### CODE REVIEW

**Review by blankdots — APPROVED** [review]

File: sda-svc/README.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ Parameter | Description | Default
 `global.confFile` | Name of config file, used when secrets are handled by hasicorp vault | `config.yaml`
 `global.confFilePath` | This path will be a subpath to the secretsPath | `""`
 `global.deploymentType` | Deployment can be split into `external` and `internal` components, available options are `all`, `external` and `internal`. | `all`
+`global.schemaType` | Standalone version requires a custom JOSN schema, availabel options are `federated` or `isolated`, unset defaults to federated. | `""`
```
Comment (pontus) [inline sda-svc/README.md:25]:
```suggestion
`global.schemaType` | Standalone version requires a custom JOSN schema, available options are `federated` or `isolated`, unset defaults to federated. | `""`
```

typo fix.

File: sda-svc/values.yaml [hunk 2]
```diff
@@ -22,6 +22,11 @@ global:
 # if left undefined or `all`, all services are deployed.
   deploymentType: "all"
 
+# standalone version requires a custom schema e.g. isolated
+# the files can still be overwritten in container with no change
+# options are `federated` or `isolated`
+  schemaType: ""
+
```
Comment (pontus) [inline sda-svc/values.yaml:29]: Clarify that the default is federated (e.g. by having it as value)?

**Review by pontus — APPROVED** [review]
Untested, looks ok. Hope this doesn't mess up #67 too much.

**Comment by blankdots** [comment]
@pontus i am using a similar setup in our fork so should work :)

**Pull request merged**